control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -3743,17 +3745,48 @@ func (r *HostedControlPlaneReconciler) reconcileOperatorLifecycleManager(ctx con
 				return fmt.Errorf("failed to reconcile catalogs: %w", err)
 			}
 
-			catalogsImageStream := manifests.CatalogsImageStream(hcp.Namespace)
-			if !overrideImages {
```

Comment:
I don't know that we want to get rid of this. I wonder if IBM uses this or there are other use cases. 🤔

Reply:
Were not getting rid of this were jus making sure it only runs if the mgmgt cluster has imageStreams capability

Reply:
Ah ok I see now